src/main/java_influxdb/com/uber/profiling/reporters/InfluxDBOutputReporter.java

```diff
@@ -0,0 +1,187 @@
+package com.uber.profiling.reporters;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.commons.lang3.StringUtils;
+import org.influxdb.BatchOptions;
+import org.influxdb.InfluxDB;
+import org.influxdb.InfluxDB.ConsistencyLevel;
+import org.influxdb.InfluxDBFactory;
+import org.influxdb.dto.BatchPoints;
+import org.influxdb.dto.Point;
+
+import com.uber.profiling.Reporter;
+import com.uber.profiling.util.AgentLogger;
+
+/**
+ * Metrics reporter class for InfluxDB. Database name "metrics" is used by this
+ * reporter. Create "metrics" database in InfluxDB before using this reporter.
+ * If you want to use different database name then update the value of "database"
+ * property in this class.
+ * 
+ * Check the "host" and "port" properties for InfluxDB and update accordingly.
+ * If Authentication is enabled in your InfluxDB then update "username" and
+ * "password" properties with appropriate values. If Authentication is not
+ * enabled then these values will not be used.
+ * 
+ * You can also pass database connection properties from yaml file and those
+ * properties will be used by this reporter.
+ * 
+ * 
+ * To uses InfluxDBOutputReporter with default database connection properties
+ * pass it in command.
+ * 
+ *     reporter=com.uber.profiling.reporters.InfluxDBOutputReporter
+ * 
+ * To use database connection properties from yaml file use below command. check
+ * sample file at "resources/influxdb.yml".
```

Comment:
I did not see file "resources/influxdb.yml"?

Reply:
Sorry, I forgot to edit the comment in java file which was copied from other branch. I don't want to include influxdb.yaml file as it will become the part of default build. This is simple file with host, port and database properties and can be created easily by users. I don't want to clutter the pom.xml with include/exclude resources.

Reply:
Sounds good then, thanks for making the change!